docs/user-guides/getting-started.md

```diff
@@ -210,14 +207,13 @@ Start mqtt:wss:external listener on 0.0.0.0:8084 successfully.
 EMQ X Broker 4.3.8 is running now
 ```
 
-2. 集群外运行本地工程项目
+2. Running project out of the cluster.
 
-> `lb` 及持久化前置依赖资源同上，需提前规划准备部署
-> 注册`crd`资源，同上
+> Prerequirements: LB, Persistence Volume, CRD
 
-* 将 `kubernetes` 集群的配置文件 `.kube/config` 复制到本地 `$HOME/.kube/config`
-* 运行 `main.go` 文件
-* 创建 `cr` 关联的 `RBAC` 及 `cr`
+* Scp the `./kube/config` of Kubernetes cluster to local `$HOME/.kube/config`
+* Run `main.go`
+* Enable the RBAC related to CR and deploy the CR
```

Comment:
Enable the RBAC related to CR and deploy the CR ->
Create RBAC objects from manifest file